src/test/java/com/techsophy/tsf/runtime/form/service/FormDataServiceElasticDisabledTest.java

```diff
@@ -1087,8 +1070,7 @@ void deleteAllFormDataByFormIdTestException()
     }
 
     @Test
-    void aggregateByFormIdFilterGroupByTest()
-    {
+    void aggregateByFormIdFilterGroupByTest() throws JsonProcessingException {
```

Comment:
Add test case to throw the exception explicitly

Reply:
Yes I will add the test case